test/blackbox-tests/test-cases/melange/intfonly.t/run.t

```diff
@@ -1,6 +1,20 @@
 Test melange libs flow when using `modules_without_implementation` stanza
 
 Build js files
-  $ dune build output/b.js
-  $ node _build/default/output/b.js
+  $ dune build @dist
+
+writing a `melange.emit` stanza in the root `dune` file overlays JS files with
+the source files (sort added because GNU find and BSD find have different
```

Comment:
Maybe use `tree` instead?

Reply:
`tree` isn't used anywhere in the tests, whereas there are multiple instances of `find`.